.github/workflows/python_release.yml

```diff
@@ -0,0 +1,30 @@
+name: Python Release
+
+on:
+#   push:
+#     tags:
+#       - "v*"
+  schedule:
+    # 00:00 UTC+8 -> 16:00
+    - cron: "0 16 * * *"
```

Comment:
How come the release is a cronjob? Do we need nightly release for Python binding?

Reply:
No need for nightly build. But I am not sure how to publish the releases properly.

Reply:
Ref https://github.com/tensorchord/llmspec/blob/main/.github/workflows/python-publish.yml

Reply:
It's triggered by the release of the main project, not the binding/python.

Reply:
There are some options:
- use `workflow_dispatch` to manually trigger it
- trigger by the main release
- trigger by the release with special tags
- a cronjob that checks the PyPI version to decide when to release a new version (complex)

I think it's okay to use the same tag as the main project. cc @VoVAllen

Reply:
The main project is a server but the bindings are clients

Reply:
And it may be tricky to handle the change of the version number. I guess that manually triggering the action is sufficient.

Reply:
Actually I think it's fine to keep it in a seperate repo like pgvector-python. The version may not be synced with pgvecto.rs itself?

Reply:
Developers may feel less convenient if spliting bindings to indenpendent repos, because of frequently switching working contexts and communication in differernt repos.

Reply:
> The version may not be synced with pgvecto.rs itself?
The version number in pyproject.toml should be edited to update the version of the output package.

Reply:
What's the final decision here? @VoVAllen @BeautyyuYanli

Reply:
I prefer to trigger it manually